app/controllers/orders_controller.rb

```diff
@@ -50,6 +50,7 @@ def order_success
       @order.received unless @order.placed_on
       @order.send_confirmation_email
       @payment.update(status: payment_status, mode: payment_type)
+      ActionCable.server.broadcast('order_success', @order.details)
```

Comment:
Write in order callback when status is changed

Reply:
Done